src/pages/Loading/Loading.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useRef } from "react";
+import Lottie from "lottie-web";
+import animationData from "../../../lib/FF_Animation_loading_03.json";
+import styles from "./Loading.module.css";
+
+function Loading() {
+  const animationElement = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    if (!animationElement.current) {
+      return;
+    }
+
+    Lottie.loadAnimation({
+      container: animationElement.current,
+      loop: true,
+      autoplay: true,
+      animationData: animationData,
+      rendererSettings: {
+        preserveAspectRatio: "xMidYMid slice",
+      },
+    });
+  }, []);
```

Comment:
Really cool effect! You can save a few lines by writing it like that: 

```suggestion
  useEffect(() => {
    if (animationElement.current) {
        Lottie.loadAnimation({
      container: animationElement.current,
      loop: true,
      autoplay: true,
      animationData: animationData,
      rendererSettings: {
        preserveAspectRatio: "xMidYMid slice",
      },
    });
    }
  }, []);
```

Reply:
Great, thanks!